Update 07-assets.md (#8449)
jekyll/jekyll: docs/_docs/step-by-step/07-assets.md

The empty front matter at the top tells Jekyll it needs to process the file. The
`@import "main"` tells Sass to look for a file called `main.scss` in the sass
directory (`_sass/`) by default which you already created directly under the root folder of your website).

At this stage you'll just have a main css file. For larger projects, this is a
great way to keep your CSS organized.

> We'll create the current class mentioned above, which will color the current link green. Create a Sass file at `_sass/main.scss` with the following content:

Reviewer: ```suggestion
Create the current class mentioned above in order to color the current link green. Create a Sass file at `_sass/main.scss` with the following content:
```
